packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -138,13 +138,13 @@ export const vModelCheckbox: ModelDirective<HTMLInputElement> = {
           assign(filtered)
         }
       } else if (isSet(modelValue)) {
-        const cloned = new Set(modelValue)
+        const updatedValue = new Set(modelValue)
         if (checked) {
-          cloned.add(elementValue)
+          updatedValue.add(elementValue)
         } else {
-          cloned.delete(elementValue)
+          updatedValue.delete(elementValue)
         }
-        assign(cloned)
+        assign(new Set(updatedValue))
```

Comment:
The change here is not necessary. 
We've already cloned the set on line 141

Reply:
@edison1105 
Thank you for review!
Obviously, it was unnecessary, I've fixed it!